packages/ckeditor5-dev-transifex/tests/createpotfiles.js

```diff
@@ -3,52 +3,78 @@
  * For licensing, see LICENSE.md.
  */
 
-'use strict';
+import { describe, expect, it, vi, beforeEach } from 'vitest';
+import createPotFiles from '../lib/createpotfiles.js';
+
+const {
+	pathJoinMock,
+	fsExistsSyncMock,
+	fsOutputFileSyncMock,
+	fsReadFileSyncMock,
+	delSyncMock,
+	findMessagesMock,
+	utilsVerifyPropertiesMock
+} = vi.hoisted( () => {
+	return {
+		pathJoinMock: vi.fn(),
+		fsExistsSyncMock: vi.fn(),
+		fsOutputFileSyncMock: vi.fn(),
+		fsReadFileSyncMock: vi.fn(),
+		delSyncMock: vi.fn(),
+		findMessagesMock: vi.fn(),
+		utilsVerifyPropertiesMock: vi.fn()
+	};
+} );
+
+vi.mock( 'path', () => {
+	return {
+		default: {
+			join: pathJoinMock
+		}
+	};
+} );
 
-const sinon = require( 'sinon' );
-const proxyquire = require( 'proxyquire' );
-const { posix } = require( 'path' );
-const { expect } = require( 'chai' );
+vi.mock( 'fs-extra', () => {
+	return {
+		default: {
+			existsSync: fsExistsSyncMock,
+			outputFileSync: fsOutputFileSyncMock,
+			readFileSync: fsReadFileSyncMock
+		}
+	};
+} );
```

Comment:
You don't need to create these mocks. Vitest handles it automatically. Example: https://github.com/ckeditor/ckeditor5-dev/blob/9efd9db5cced5ad4d68028138ceb3eb94f299637/packages/ckeditor5-dev-utils/tests/tools/getdirectories.js.

I propose simplifying it by using `vi.mocked( fn )` instead of defining everything manually.